.rubocop.yml

```diff
@@ -28,6 +28,12 @@ Rails:
 Lint/DeprecatedOpenSSLConstant:
   Enabled: true
 
+Lint/MissingSuper:
+  Enabled: true
+  Exclude:
+    - 'app/components/**/*'
+    - 'config/initializers/okcomputer.rb'
+
```

Comment:
moved this one configuration to alphabetize our localized configurations for cops